src/components/Form/TalentLayerIdForm.tsx

```diff
@@ -62,7 +62,7 @@ function TalentLayerIdForm() {
           const response = await delegateMintID(
             chainId,
             submittedValues.handle,
-            handlePrice,
+            Number(handlePrice),
```

Comment:
use BigInt, never Number for wei value

Reply:
I made it to number only for api call

Its bigint when i sent made transaction
Its number for making api request

If i give bigint for api request i am getting error as unable to serialize bigint